src/main/java/org/opentripplanner/updater/alerts/GtfsEnhancedRealtimeAlertsUpdater.java

```diff
@@ -77,21 +77,31 @@ public String toString() {
         JsonNode json = mapper.readTree(jsonString);
 
         for (JsonNode node: json.get("entity")) {
+            if (!node.hasNonNull("alert")) {
+                continue;
+            }
+
+            JsonNode alert = node.get("alert");
+
+            if (!alert.hasNonNull("informed_entity")) {
```

Comment:
Similarly here, maybe just simply use `get` and check for `null`. If not, then perhaps you could use the `has` method to avoid the double negative?

Reply:
Sure, done